### DISCUSSION TITLE
Authenticator is blank

### DISCUSSION THREAD

**Discussion by timmya3** [Authenticator-Extension/Authenticator#695] [Q&A]
I added a second MS account on my computer and was receiving two codes when I selected authenticator. I needed to remove that so I removed the extension (now I know I should not of done that) from chrome. When I re-added it Authenticator is now blank with no codes. I added the second person to a new computer and when I hit authenticator it is also blank? Is there a way to re-add the QR codes for me and the other user?

**Comment by Sneezry** [comment]
> Is there a way to re-add the QR codes for me and the other user?

You need recover data with a backup.

**Reply by waveformula** [reply]
What is I don't have a backup?

**Reply by waveformula** [reply]
*if

**Comment by dooleydiligent** [comment]
I have the same blank authenticator, only I haven't reinstalled anything.  I just needed to use it, and it was all but gone.  I can get to "Settings" and I can click the "Edit" and "Add Account" buttons, but the display itself does not change.  Any reset steps are appreciated.

**Reply by mymindstorm** [reply]
If you need to reset, go to manage extensions, click details on Authenticator, and then click on extension options.
